dropwizard-jersey/src/main/java/io/dropwizard/jersey/validation/FuzzyEnumParamConverterProvider.java

```diff
@@ -39,6 +42,10 @@
         final Class<Enum<?>> type = (Class<Enum<?>>) rawType;
         final Enum<?>[] constants = type.getEnumConstants();
         final String parameterName = getParameterNameFromAnnotations(annotations).orElse("Parameter");
+        final Method fromStringM = Stream.of(type.getMethods())
+            .filter(m -> m.getName().equals("fromString"))
+            .findFirst()
+            .orElse(null);
```

Comment:
I was looking for a way to get this safely `type.getMethod("fromString", String.class)` without throwing a `NoSuchMethodException`